packages/cli/src/runAndroid/runAndroid.js

```diff
@@ -33,7 +34,7 @@ function checkAndroid(root) {
  */
 function runAndroid(argv: Array<string>, config: ContextT, args: Object) {
   if (!checkAndroid(args.root)) {
-    console.log(
+    logger.info(
       chalk.red(
```

Comment:
this should be `logger.error` without chalk

Reply:
Fixed